vendor/engines/split_accounts/app/models/split.rb

```diff
@@ -0,0 +1,19 @@
+class Split < ActiveRecord::Base
```

Comment:
I could go for `SplitAccounts::Split`.  I started off with `AccountSplit` and it got confusing with the joins.